libs/agent-sm/agent/src/main/java/org/opensearch/javaagent/Agent.java

```diff
@@ -55,12 +57,32 @@ public static void agentmain(String agentArguments, Instrumentation instrumentat
 
     private static AgentBuilder createAgentBuilder(Instrumentation inst) throws Exception {
         final Junction<TypeDescription> systemType = ElementMatchers.isSubTypeOf(SocketChannel.class);
+        final Junction<TypeDescription> pathType = ElementMatchers.isSubTypeOf(Files.class);
+        final Junction<TypeDescription> fileChannelType = ElementMatchers.isSubTypeOf(FileChannel.class);
 
-        final AgentBuilder.Transformer transformer = (b, typeDescription, classLoader, module, pd) -> b.visit(
+        final AgentBuilder.Transformer socketTransformer = (b, typeDescription, classLoader, module, pd) -> b.visit(
             Advice.to(SocketChannelInterceptor.class)
                 .on(ElementMatchers.named("connect").and(ElementMatchers.not(ElementMatchers.isAbstract())))
         );
 
+        final AgentBuilder.Transformer fileTransformer = (b, typeDescription, classLoader, module, pd) -> b.visit(
+            Advice.to(FileInterceptor.class)
+                .on(
+                    ElementMatchers.named("delete")
+                        .or(ElementMatchers.named("deleteIfExists"))
```

Comment:
Can we use ElementMatchers.namedOneOf here and provide the list in instead of a big or block?